docs/superpowers/plans/2026-04-09-modular-content-skills.md

```
@@ -0,0 +1,1536 @@
+```
+Score: [X]/100 ([Rating])
+Checked: [date]
+
+PASS: [list items that passed]
+ISSUES:
+- [Item]: [specific problem + fix recommendation]
+- [Item]: [specific problem + fix recommendation]
+
+Suggested metaTitle: "[if missing or bad, suggest one]"
+Suggested metaDescription: "[if missing or bad, suggest one]"
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)